### PR TITLE
remove null tests for person_id for models built from event based grains

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:** #37 

**This PR will result in the following new package version:**

v0.7.2

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

- Removes `not_null` tests for `person_id` from select models. This is because Klaviyo can record events with deactivated profiles, resulting in null `person_id`s. Therefore models built off of tables with event-based grains may have null `person_id`s.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [ ] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [ ] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

- tests now pass and we see they have been removed from docs

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [na] README updates have been applied (if applicable)
    
- [na] DECISIONLOG updates have been updated (if applicable)
- [na] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [x] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# dbt_klaviyo v0.7.2
+[PR #38](https://github.com/fivetran/dbt_klaviyo/pull/38) includes the following updates:
+
+## Bug Fixes
+- Removes `not_null` tests for `person_id` from select models. This is because Klaviyo can record events with deactivated profiles, resulting in null `person_id`s. Therefore models built off of tables with event-based grains may have null `person_id`s.
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:5]: Can we list all the models this test was removed from. This way it is clear to users upgrading which models this is impacting directly.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-reneeli this looks great and thanks for applying these updates! I have one comment to update the CHANGELOG to be more explicit around which of the models this test update is impacting. 

Once that is done, be sure to regen the docs and then this is good to kick off the release process. Also, this package does not have the auto releaser yet. So feel free to either include it or remember that it will need to be manually generated.

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-reneeli) [reply CHANGELOG.md]: done! Also did docs regen and made sure the tests have been removed. And added the auto releaser. once buildkite passes I'll kick off the release process

**Review by fivetran-catfritz — APPROVED** [review]
lgtm

**Pull request merged**